src/PageSections/_Private/stringify_generic.hack

```diff
@@ -23,15 +23,18 @@ function stringify_generic(
   } else {
     $base = '';
   }
-  $base .= ns_normalize_type($ns, $generic->getName());
+  $base .= $generic->getName();
```

Comment:
Pretty sure this was a bug, we were normalizing the generic's name instead of its constraint.